docs/source/en/create_a_model.md

```diff
@@ -333,16 +333,6 @@ config = MaskFormerConfig(backbone="microsoft/resnet50", use_pretrained_backbone
 model = MaskFormerForInstanceSegmentation(config) # head
 ```
 
-You could also load the backbone config separately and then pass it to the model config.
```

Comment:
Removed as it's repeated and included in the section about loading pretrained backbones, but the example loads randomly initialized weights